test/Microsoft.AspNetCore.Mvc.Razor.ViewCompilation.FunctionalTests/PublishWithEmbedViewSourcesTest.cs

```diff
@@ -32,9 +32,9 @@ public PublishWithEmbedViewSourcesTest(PublishWithEmbedViewSourcesTestFixture fi
             // Arrange
             var expectedViews = new[]
             {
-                "Areas/TestArea/Views/Home/Index.cshtml",
-                "Views/Home/About.cshtml",
-                "Views/Home/Index.cshtml",
+                "/Areas/TestArea/Views/Home/Index.cshtml",
```

Comment:
The absence of the first slash makes the view engine treat this is as a relative path (relative to the current view \ app root when there is no view), as opposed to an app-relative path which is what this is (https://github.com/aspnet/Mvc/blob/dev/src/Microsoft.AspNetCore.Mvc.Razor/RazorViewEngine.cs#L530-L534).